### PR TITLE
added pyvisa-py from pypi

### PR DESCRIPTION
**Pull request by greyltc**
- need to work out how to add optional dependencies: pyUSB and pyserial
- need to solve import error in testing step (module name can't contain hyphen)

### CODE REVIEW

**Comment by greyltc** [comment]
I'm seeing
`/Users/travis/miniconda/envs/_build_placehold_placehold_placehold_placehold_plac/bin/python: No such file or directory`
in the travis-ci logs. Is that something wrong with my PR here or is the CI robot somehow broken?

**Comment by melund** [comment]
@greyltc You can ping the @conda-forge/core team to get some help why this is failing.

File: recipes/pyvisa-py/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```
Comment (ocefpaf) [inline recipes/pyvisa-py/bld.bat]: You do not need the `bld.bat`.

File: recipes/pyvisa-py/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (ocefpaf) [inline recipes/pyvisa-py/build.sh]: You also do not need the `build.sh`.

File: recipes/pyvisa-py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+package:
+  name: pyvisa-py
+  version: "0.2"
```
Comment (ocefpaf) [inline recipes/pyvisa-py/meta.yaml]: Please check the example to improve this recipe maintainability. We prefer to have at least the version as a `jinja2` variable for easier updates.

File: recipes/pyvisa-py/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+source:
+  fn: PyVISA-py-0.2.tar.gz
+  url: https://pypi.python.org/packages/59/ef/c79cd2c221dba12628242e917afe53aa08bb221016f7bee76ea1ff130a22/PyVISA-py-0.2.tar.gz
```
Comment (ocefpaf) [inline recipes/pyvisa-py/meta.yaml]: Check the example on how to use a more readable URL from `pypi.io`.

File: recipes/pyvisa-py/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  md5: 1f5433d4f0b8f42bd8507fbc0a77f66b
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  noarch_python: True
+  # preserve_egg_dir: True
```
Comment (ocefpaf) [inline recipes/pyvisa-py/meta.yaml]: Please remove all boilerplate comments.

File: recipes/pyvisa-py/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    # syntax is module:function.  For example
+    #
+    # - pyvisa-py = pyvisa-py:main
+    #
+    # Would create an entry point called pyvisa-py that calls pyvisa-py.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
```
Comment (ocefpaf) [inline recipes/pyvisa-py/meta.yaml]: Add here a line with:

```yaml
script: python setup.py install --single-version-externally-managed --record record.txt
```

**Comment by melund** [comment]
@greyltc You can find the recipe example here: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

File: recipes/pyvisa-py/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - pyvisa
+
+  commands:
+    - python -c "import pyvisa.testsuite as ts; ts.run()"
+
+about:
+  home: https://github.com/hgrecco/pyvisa-py
+  license: MIT
```
Comment (jakirkham) [inline recipes/pyvisa-py/meta.yaml]: Please add `license_file: LICENSE` so as to include the license in the `conda` package.

**Comment by scopatz** [comment]
Please enable CircleCI on your fork of staged recipes so that this builds on linux, http://conda-forge.github.io/docs/ciservices.html#circleci-linux

**Comment by greyltc** [comment]
Hi @scopatz I looked into doing this as @ocefpaf mentioned previously. It looks to me like CircleCI links with my Github account and requires access to _all_ my repos, public and private. That's unfortunately not a permission I'm willing to grant. What are our alternatives here?

Is my only option to make a new dummy github account and transfer my fork there?

**Comment by ocefpaf** [comment]
> Hi @scopatz I looked into doing this as @ocefpaf mentioned previously. It looks to me like CircleCI links with my Github account and requires access to _all_ my repos, public and private. That's unfortunately not a permission I'm willing to grant. What are our alternatives here?

Activating `CircleCI` is pretty safe and higly desirable, but  we cannot ask you do that against your will. I tested this locally on Linux and it is passing, so I am merging this. If a dummy account is OK for you we would be fine with that for future interactions.

Thanks @greyltc!

**Comment by greyltc** [comment]
Thanks @ocefpaf!

**Pull request merged**